core-api/src/main/java/com/optimizely/ab/config/ProjectConfig.java

```diff
@@ -275,18 +271,19 @@ public Experiment getExperimentForKey(@Nonnull String experimentKey,
     /**
      * Helper method to retrieve the {@link EventType} for the given event name.
      * If {@link RaiseExceptionErrorHandler} is provided, either an event type is returned,
-     *  or an exception is sent to the error handler if there are no event types in the project config with the given name.
+     * or an exception is sent to the error handler if there are no event types in the project config with the given name.
      * If {@link NoOpErrorHandler} is used, either an event type or {@code null} is returned.
      *
-     * @param eventName the event type to retrieve from the current project config
+     * @param eventName    the event type to retrieve from the current project config
      * @param errorHandler the error handler to send exceptions to
      * @return the event type for the given event name
      */
-    public @CheckForNull EventType getEventTypeForName(String eventName, ErrorHandler errorHandler) {
+    public @CheckForNull
+    EventType getEventTypeForName(String eventName, ErrorHandler errorHandler) {
```

Comment:
Please move the `@CheckForNull` annotation to before `public` like how its used  on `getExperimentForKey`. They are equivalent and wont produce weird formatting where `public` is on different line. That's easy to misread as a package-local visibility.